docs/source/references/data_resource_schema.rst

```diff
@@ -0,0 +1,8 @@
+.. _data-resource-schema:
+
+Data Resource Schema Reference
```

Comment:
I think this page should be consolidated with Basic Usage (or "Creating and Using a Data Resource Schema"). See my other comment.